students/TrkCat/Lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,10 @@
+for num in range(1:101):
```

Comment:
`range(1:101)` is an invalid syntax, should be `range(1, 101)`

Reply:
Could have sworn I checked this code through, but looks like I made an (incorrect) update afterwards. Thanks much, will update.